andela/ah-kgl-avengers-frontend
# #165020163 User should be able to reset password

### What does this PR do?
Give a user functionality to change Password
### Description of Task to be completed?
- User should request to reset password
- User should receive an email with a link to reset password
- User should reset his/her password
### How should this be manually tested?
- Clone the repository locally with `git clone https://github.com/andela/ah-kgl-avengers-frontend.git` command.
- Check out to this branch by running `git checkout ft-user-reset-password-165020163`.
- Install dependencies by running `npm install`.
- Update the .env file according to the .env.sample file
- Start the server with `npm start` and put this url `http://localhost:8000/reset` in the address bar of your browser then follow the instruction of resetting your password.
### Any background context you want to provide?
- User should have an account on `AuthorsHaven` to be able to reset an account.
### What are the relevant pivotal tracker stories?
[#165020163](https://www.pivotaltracker.com/n/projects/2321840/stories/165020163)
### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29593858/59760630-07c96e00-9293-11e9-93dc-d2a850108260.png)
![image](https://user-images.githubusercontent.com/29593858/59760642-0f891280-9293-11e9-8bef-5844b4b63775.png)
![image](https://user-images.githubusercontent.com/29593858/59760658-17e14d80-9293-11e9-915f-d2e6615db488.png)

### Questions:
N/A
